Jenkinsfile

```diff
@@ -74,10 +74,13 @@ def commonTestUbuntu = {
 
 def buildDebUbuntu = { repoName, releaseVersion, sourcePath ->
     def volumeName = "$name-deb-u1604"
-    sh "docker volume rm -f $volumeName"
+    if ("${BRANCH_NAME}" != '' && "${BRANCH_NAME}" != 'master') volumeName = "${volumeName}.${BRANCH_NAME}"
```

Comment:
Please, use multiline with curly braces for more readability.
As an option bash-like "${BRANCH_NAME}" in conditional could be replaced with env.BRANCH_NAME.